pkg/collector/json_path_collector.go

```diff
@@ -131,3 +143,82 @@ func getPodMetrics(pod *corev1.Pod, scheme, path string, port int) ([]byte, erro
 
 	return data, nil
 }
+
+// intsToFloat64s will convert a slice of int to a slice of float64
+func intsToFloat64s(in []int) (out []float64) {
+	out = []float64{}
+	for _, v := range in {
+		out = append(out, float64(v))
+	}
+	return
+}
+
+// float32sToFloat64s will convert a slice of float32 to a slice of float64
+func float32sToFloat64s(in []float32) (out []float64) {
+	out = []float64{}
+	for _, v := range in {
+		out = append(out, float64(v))
+	}
+	return
+}
+
+// reduce will reduce a slice of numbers given a reducer function's name. If it's empty or not recognized, avg is used.
+func reduce(values []float64, aggregator string) (float64, error) {
```

Comment:
nit: the comment here is now a bit outdated after the name change to `aggregator`

Reply:
fixed! :D

Reply:
The default also changed from `avg` to `error` right? :)

Reply:
Sorry for nitpicking. I think it's the only thing, then it's gtg from my side.

Reply:
No worries, it's my fault :)

Reply:
Fixed in last commit